resources/guides/ta_guide.md

```diff
@@ -0,0 +1,99 @@
+---
+id: teaching assistant
+title: How To Become a Teaching Assistant
+sidebar_label: How To Become a Teaching Assistant
+slug: /guides/ta_guide
+---
+
+_Last updated on 2023/09/07_
+
+## What is a TA?
+
+A TA is a unionized student position where students can work with instructors to run University of Windsor courses. Instructors could ask for a TA's help in a variety of ways:
+
+- Holding Office Hours 
+- Grade student assessments
+- Proctor Exams
+- Taking notes or answering questions during classes
+    
+A TA refers to an undergraduate student while GA (Graduate Assistant) refers to Graduate students
```

Comment:
Since you defined "GA", you should also define TA. Also a good idea since this is one of the first usages of the acronym anyway.